src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/DependenciesInitializer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using System.IO;
+using System.Reflection;
+using Uno.Extensions;
+using Uno.Extensions.Specialized;
+using Uno.Roslyn;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+
+#if NETFRAMEWORK
+using Uno.SourceGeneration;
+#endif
+
+namespace Uno.UI.SourceGenerators
+{
+	internal class DependenciesInitializer
+	{
+		internal static void Init(GeneratorExecutionContext context)
+		{
+			// Debugger.Launch();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270956)